src/CQRS/LeanCode.CQRS.AspNetCore/Local/ILocalCommandExecutor.cs

```diff
@@ -0,0 +1,10 @@
+using LeanCode.Contracts;
+using Microsoft.AspNetCore.Http;
+
+namespace LeanCode.CQRS.AspNetCore.Local;
+
+public interface ILocalCommandExecutor
+{
+    Task<CommandResult> RunAsync<T>(HttpContext context, T command, CancellationToken cancellationToken = default)
```

Comment:
What if we don't have any http context at hand? Should the caller create their own fake one? I think at minimum we'd just need DI scope and claims principal to run a command, the rest might be 'faked' for the sake of local execution.

Reply:
> we'd just need DI scope and claims principal

`ClaimsPrincipal` will be enough I think. DI scope should not be controlled from the outside, because each call _must_ open a new scope either way (otherwise a lot of things will break).

Also, the more I think about it, the more I am in favor of **not** exposing the `HttpContext` overload. Mocking everything is just _more_ correct, and even giving access to `Request` is wrong (because that is completely different request than the one being run). /cc: @lukaszgarstecki, I said something different earlier.

Reply:
Also, implementing `Request` and `Response` as just "return empty data and ignore writes" is more and more OK with me.

Reply:
Done, no `HttpContext` in sight.